### DISCUSSION TITLE
Solidus 2.11.17 does not support GBP currency

### DISCUSSION THREAD

**Discussion by keferboeck** [solidusio/solidus#4523] [Troubleshooting]
**Solidus Version**:
2.11.17

**To Reproduce**

_/initializers/spree.rb_
`config.currency = "GBP"`
(instead of USD)

In the Database, run the following queries to (against the sample store) to switch from USD to GBP

```
update spree_orders
set currency = "GBP"
where currency = "USD";


update spree_prices
set currency = "GBP" 
where currency = "USD";


update spree_variants
set cost_currency = "GBP" 
where cost_currency = "USD";
```

**Current behavior**

When currency is set to GBP, the checkout process does throw an error past the "address" step. The error message is: 
`"We are unable to calculate shipping rates for the selected items."`

**Expected behavior**

Once the address fields are correctly field in, clicking "save & continue" should take you to the next step "Delivery".

**Screenshots**

**Desktop (please complete the following information):**
 - MacOS Big Sur (Version: 11.2.2)
 - Browser: Chrome Version 104.0.5112.101 (Official Build) (arm64)

**Additional context**

I tested the behaviour against the sample store, as well as products we added. That shop was build from scratch without any sample data. The error appears on both setups. I backed up the databases if that is of any help, but I am sure you can easily reproduce it yourself.

**Comment by jarednorman** [comment]
You need to update the shipping methods as well, as they are all likely configured for USD.

If you have further issues with this, please open a discussion in the [Troubleshooting section](https://github.com/solidusio/solidus/discussions/categories/troubleshooting) or ask on Slack.

**Reply by kennyadsl** [reply]
@jarednorman @keferboeck I converted this issue into a discussion, so you can mark an answer as the right one and help future people in the same trouble to find an answer easier. Thanks to both!

**Reply by keferboeck** [reply]
@jarednorman  - there is no reference to currency in any of the shipping methods tables. The only time currency appears within shipping tables is in spree_shipping_methods.name 

Example: UPS Ground (USD)

I would be surprised if the name field's content in the brackets (USD) makes the difference?!

Or perhaps, can you please be a little more specific about where precisely I can change the currency in shipping methods? Thank you!

**Reply by jarednorman** [reply]
It's in the preferences on the shipping method's calculator.
